### PR TITLE
Feature/conditionally show electrical tags

### PR DESCRIPTION
**Pull request by Noggling**
# Description

3d to piping-and-ht app and elektro tags toggle button. 
all implemented behind !isProducion to be able to release to dev and test

Completes: [AB#200353](https://dev.azure.com/Equinor/fa63ceea-8883-41e2-8823-a3b54e4be178/_workitems/edit/200353)  [AB#214094](https://dev.azure.com/Equinor/fa63ceea-8883-41e2-8823-a3b54e4be178/_workitems/edit/214094)

## Checklist:

-   [x] I have performed a self-review of my own code.
-   [x] I have linked my DevOps task using the AB# tag.
-   [x] My code is easy to read, and comments are added where needed.
-   [ ] My code is covered by tests.

### CODE REVIEW

File: src/apps/DisciplineReleaseControl/Components/3D/3dView.tsx [hunk 1]
```diff
@@ -0,0 +1,86 @@
+import { tokens } from '@equinor/eds-tokens';
+import { Icon } from '@equinor/lighthouse-components';
+import { useModelViewerContext, Viewer } from '@equinor/lighthouse-model-viewer';
+import { useFacility } from '@equinor/lighthouse-portal-client';
+import { useEffect, useMemo, useState } from 'react';
+import { useQuery } from 'react-query';
+import { ElectroIcon } from '../../../../packages/ModelViewer/icons/ElectroIcon';
```
Comment (jaysencpp) [inline src/apps/DisciplineReleaseControl/Components/3D/3dView.tsx:7]: import { ElectroIcon } from '../../../../packages/ModelViewer/icons/ElectroIcon';
Har vi alias på denne?

Comment (Noggling) [reply src/apps/DisciplineReleaseControl/Components/3D/3dView.tsx:7]: bør legges i en annen plass tenker jeg ;)

File: src/packages/ModelViewer/context/modelViewerContext.tsx [hunk 2]
```diff
@@ -110,30 +119,50 @@ export const ModelViewerContextProvider = ({
                 if (!plantState.tags) return;
                 await selection.setSelectionBasedOnE3dTagNos(plantState.tags);
 
+                // setState((s) => ({
```
Comment (jaysencpp) [inline src/packages/ModelViewer/context/modelViewerContext.tsx:122]: kommentert ut kode

File: src/apps/DisciplineReleaseControl/Components/Electro/ElectroView.tsx [hunk 3]
```diff
@@ -29,24 +29,8 @@ export const ElectroView = ({
...
-        circuitStarterTagNos = pipetest.circuits[0].circuitAndStarterTagNo;
-        circuitStarterTagNosArray.push(pipetest.circuits[0].circuitAndStarterTagNo);
-        for (let i = 1; i < pipetest.circuits.length; i++) {
-            circuitStarterTagNos = circuitStarterTagNos.concat(
-                ',' + pipetest.circuits[i].circuitAndStarterTagNo
-            );
-            circuitStarterTagNosArray.push(pipetest.circuits[i].circuitAndStarterTagNo);
-        }
-    }
+    const circuitStarterTagNosArray = pipetest.circuits?.map((c) => c.circuitAndStarterTagNo);
```
Comment (jaysencpp) [inline src/apps/DisciplineReleaseControl/Components/Electro/ElectroView.tsx:32]: prøvde å forstå det som ble fjernet her, er de to linjene med kode du la til det samme som det som ble fjernet? 🤔

File: src/packages/ModelViewer/context/modelViewerContext.tsx (hunk 2 above)
Comment (Noggling) [reply src/packages/ModelViewer/context/modelViewerContext.tsx:122]: det sletter jeg ;)

**Pull request merged**